src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -450,46 +392,25 @@ public void DirectorySecurity_CreateDirectory_InvalidPath()
         }
 
         [Fact]
-        public void DirectorySecurity_CreateDirectory_DefaultDirectorySecurity()
-        {
-            DirectorySecurity security = new DirectorySecurity();
-            Verify_DirectorySecurity_CreateDirectory(security);
-        }
-
-        [Theory]
-        [InlineData(FileSystemRights.ReadAndExecute, AccessControlType.Allow)]
-        [InlineData(FileSystemRights.ReadAndExecute, AccessControlType.Deny)]
-        [InlineData(FileSystemRights.WriteData,      AccessControlType.Allow)]
-        [InlineData(FileSystemRights.WriteData,      AccessControlType.Deny)]
-        [InlineData(FileSystemRights.FullControl,    AccessControlType.Allow)]
-        [InlineData(FileSystemRights.FullControl,    AccessControlType.Deny)]
-        public void DirectorySecurity_CreateDirectory_DirectorySecurityWithSpecificAccessRule(
-            FileSystemRights rights,
-            AccessControlType controlType)
+        public void DirectorySecurity_CreateDirectory_DirectorySecurityWithSpecificAccessRule()
         {
-            DirectorySecurity security = GetDirectorySecurity(rights, controlType);
-            Verify_DirectorySecurity_CreateDirectory(security);
-        }
-
-        [Fact]
-        public void DirectorySecurity_CreateDirectory_DirectoryAlreadyExists()
```

Comment:
Seems like you are merging this test with `DirectorySecurity_CreateDirectory_DirectorySecurityWithSpecificAccessRule`. Can you keep them separate or is there any reason for that?

Reply:
My mistake. The name of this test should be `DirectorySecurity_CreateDirectory_DirectoryAlreadyExists`.